.buildkite/premerge.definition.yaml

```
@@ -0,0 +1,12 @@
+agent_queue_id: trigger-pipelines
+description: Build Unreal GDK
+github:
+  branch_configuration:
```

This will need to change before merging, since it restricts CI such that it will only run against this branch.  I think you probably want to set this to `branch_configuration: []` and remove the specified branch to make it run on all branches.
```suggestion
  branch_configuration: []
  
```